src/app/content/highlights/utils/paginationUtils.ts

```diff
@@ -35,13 +35,13 @@ const totalOfCountsPerSource: (counts: CountsPerSource) => number = flow(
 export const getNextPageSources = (
   remainingCounts: CountsPerSource,
   tree: ArchiveTree,
-  nextPageSize: number
+  nextPageSize?: number
```

Comment:
you don't want this to return too many sources, because you'll start getting errors from the api that the url is malformed. in the case of the print we do want to target loading all highlights so having a null/undefined `nextPageSize` makes sense, but we should also introduce a constant for `max_query_sources` or something and limit the number of potential sources to return here